com.bitwarden.desktop.yml

```diff
@@ -0,0 +1,79 @@
+app-id: com.bitwarden.desktop
+base: org.electronjs.Electron2.BaseApp
+base-version: stable
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: bitwarden
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: nodejs
+    cleanup: 
+      - /include
+      - /share
+      - /app/lib/node_modules/npm/changelogs
+      - /app/lib/node_modules/npm/doc
+      - /app/lib/node_modules/npm/html
+      - /app/lib/node_modules/npm/man
+      - /app/lib/node_modules/npm/scripts
+    sources:
+      - type: archive
+        url: https://nodejs.org/dist/v8.11.3/node-v8.11.3.tar.xz
+        sha256: 577c751fdca91c46c60ffd8352e5b465881373bfdde212c17c3a3c1bd2616ee0
+  - name: libsecret
```

Comment:
insert newlines between modules: https://github.com/flathub/flathub/wiki/YAML-Style-Guide

Reply:
Oh, I was not aware we had an official style guide.

Thanks for pointing it out :-)